app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -473,18 +473,20 @@ class SamplesView extends React.Component {
   };
 
   handleBulkDownloadModalOpen = () => {
-    const { maxSamplesBulkDownload, admin } = this.context || {};
-    if (!maxSamplesBulkDownload) {
+    const { appConfig, admin } = this.context || {};
+    if (!appConfig.maxSamplesBulkDownload) {
       this.setState({
         bulkDownloadButtonTempTooltip:
           "Unexpected issue. Please contact us for help.",
       });
     } else if (
-      this.props.selectedSampleIds.size > parseInt(maxSamplesBulkDownload) &&
+      this.props.selectedSampleIds.size > appConfig.maxSamplesBulkDownload &&
```

Comment:
please add a comment here that `maxSamplesBulkDownloadOriginalFiles` is handled later by disabling the option